submissions.html

```diff
@@ -0,0 +1,149 @@
+
+</ol>
+
+
+<p class='font-medium'>Paper Submission</p>
+<!-- #E40066 color for when submissions are closed. -->
+<p>The paper submission for ISQCMC '25 is currently <a href="mailto:quantummusicsubmission@gmail.com"><span style='color: #03CEA4;'>open</span></a>. Please refer to the detailed instructions below to ensure a smooth peer-review and avoid unnecessary rejections of your submission.</p>
+
+
+The deadline for submission is <strong>May 09, 2025</strong>. <br>
+
+<p>Paper Submission Instructions</p>
+Please make sure the paper is formatted according to the <a href="https://www.overleaf.com/latex/templates/isqcmc-2025-template/qyfngrjtyvjv"><u>ISQCMC 2025 Template</u></a>, is <em> anonymized</em> (see the Anonymization section below),and does not exceed 5 pages.<br><br>
+
+All submissions are to be sent to <a href="mailto:quantummusicsubmission@gmail.com">quantummusicsubmission@gmail.com</a>, and <strong> should contain the following information and files:</strong>
+
+<ul>
+  <li>Full names of all authors, and their respective affiliations (if applicable)</li>
+  <li>Clear indication of the corresponding author(s) and their contact email(s)</li>
+  <li>The <em>anonymized version</em> of your paper, in PDF format</li>
+  <li>Any additional supporting information or appendices</li>
+</ul>
+  
+
+
+
+<p> Anonymization </p>
+Your paper should be anonymized for the double-blind peer-review process. This means that you should hide or censor any information that could identify the authors, such as names, affiliations, or acknowledgments. Any other information that may reveal your identity should be ideally avoided. In case you are using a substantial amount of self-referential citations, you may also consider censoring them while adjusting the writing to imply so (e.g. "In our previous work [censored citation]"). <br>
+
+<p><em>LaTeX Template Anonymization usage:</em></p> The <a href="https://www.overleaf.com/latex/templates/isqcmc-2025-template/qyfngrjtyvjv"><u>ISQCMC 2025 LaTeX Template</u></a> has a feature that submitters can use to anonymize their papers. Its usage is explained below. <br><br>
+
+Check <strong>line 20</strong> of the template for the line reading <tt>\usepackage[blind]{anonymize}</tt><br>
+
+This is the command that imports the anonymization functionality, and allows the submitter to censor content by wrapping the text around the <tt>\anonymize{ANY CONTINUOUS TEXT HERE}</tt> environment. Notice that in the template, parts of the abstract and the sponsorship footnote were written this way.
+
+<strong> Switch the anonymization on/off:</strong> While writing the paper, you can switch the anonymization off by removing the <tt>[blind]</tt> argument from the command: <tt>\usepackage{anonymize}</tt>.
+
+<br><br> In case there are any questions, please do not hesitate to reach out to the organizers through our email: <a href="mailto:quantummusicsubmission@gmail.com">quantummusicsubmission@gmail.com</a>
```

Comment:
again please consider non literal emails on the web.

Reply:
Please check [a5d7055](https://github.com/isqcmc/isqcmc2025/pull/31/commits/a5d705515f00bf846fcbea92c64260bd3b894acf)